TriangleApp/app/src/main/java/triangle/triangleapp/activities/CameraActivity.java

```diff
@@ -0,0 +1,46 @@
+package triangle.triangleapp.activities;
+
+import android.hardware.Camera;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.FrameLayout;
+
+import triangle.triangleapp.helpers.CameraHelper;
+import triangle.triangleapp.view.CameraPreview;
+import triangle.triangleapp.R;
+import triangle.triangleapp.helpers.MediaHelper;
+
+/**
+ * Created by Kevin Ly on 6/12/2017.
+ */
+
+public class CameraActivity extends AppCompatActivity{
+    private Button btnCapture;
```

Comment:
Modified to local variable.
Button btnCapture = (Button) findViewById(R.id.btnCapture);